crates/database/src/types/documents.rs

```diff
@@ -79,11 +79,7 @@ impl DemotionDocument {
     }
 }
 
-#[derive(Debug, Serialize, Deserialize)]
-pub struct BuilderInfoDocument {
-    pub pub_key: BlsPublicKey,
-    pub builder_info: BuilderInfo,
-}
+pub type BuilderInfoDocument = BuilderConfig;
```

Comment:
Not a huge fan of the type aliasing done here but this greatly reduces the amount of changes and avoids circular dependencies with the `common` crate.